scenarios/centos-9/ironic.yml

```diff
@@ -1,25 +1,31 @@
 ---
 cifmw_install_yamls_vars:
   BMO_SETUP: false
-  STORAGE_CLASS: crc-csi-hostpath-provisioner
   INSTALL_CERT_MANAGER: false
 
-cifmw_edpm_prepare_skip_crc_storage_creation: true
-
-pre_deploy:
-  - name: Kustomize OpenStack CR
-    type: playbook
-    source: control_plane_ironic.yml
-
-cifmw_services_ironic_enabled: true
+cifmw_edpm_prepare_kustomizations:
```

Comment:
It is a cleaner approach, but note that it will not work for all components, if this was the original idea.
we have components that needs more than a kustomization, and in that case, a extra hook is still needed. E.g: Ceph scenarios, where kustomization depends on ceph playbook results.

Reply:
Thank you @viroel for your input. Let me add back the kustomize and keep this option for end user.